packages/core/admin/package.json

```diff
@@ -55,26 +54,27 @@
     "@strapi/helper-plugin": "4.2.3",
     "@strapi/icons": "1.2.0",
     "@strapi/utils": "4.2.3",
-    "axios": "0.24.0",
+    "axios": "0.25.0",
     "babel-loader": "8.2.5",
     "babel-plugin-styled-components": "2.0.2",
     "bcryptjs": "2.4.3",
-    "chalk": "^4.1.1",
-    "chokidar": "^3.5.1",
+    "chalk": "4.1.2",
+    "chokidar": "3.5.3",
     "codemirror": "^5.65.6",
-    "cross-env": "^7.0.3",
+    "cropperjs": "1.5.12",
     "css-loader": "6.7.1",
     "date-fns": "2.28.0",
-    "dotenv": "8.5.1",
+    "dotenv": "14.2.0",
+    "duplicate-dependencies-webpack-plugin": "^1.0.2",
     "esbuild-loader": "^2.19.0",
-    "execa": "^1.0.0",
+    "execa": "5.1.1",
     "fast-deep-equal": "3.1.3",
     "font-awesome": "^4.7.0",
-    "formik": "^2.2.6",
-    "fs-extra": "10.0.0",
+    "formik": "^2.2.9",
```

Comment:
I agree that things are a bit up in the air at the moment, I basically stuck with what was most used in each case - and kept consistency, even when it wouldn't resolve to another version - but I planned on having a chat about this, We should have it now honestly, cc: @alexandrebodin.

It's a complex topic 🙃

Pros of pinning dependencies:
- We basically avoid any surprises if any of our dependencies decide that they want to break semver and introduce breaking changes (or just introduce a bug in a minor a patch release, nobody's perfect).
Keep in mind that we would not be bulletproof anyway as we could still be impacted by issues like this if our dependencies themselves do not pin theirs, this security stops on the first layer that's what I'm saying.

Cons:
- As package authors we want to could want to use wider ranges to prevent duplication in the downloaded modules of our users. Pinning messes up with that. It's definitely less of a problem in a node environment.
- Security fixes released in our dependencies needs to wait for the next release for them to be available - easily - to our users.
- A bit of upgrade noise in the repo related to the PRs upgrading our dependencies.

Reply:
Unrelated but we could use [that feature of yarn v3](https://yarnpkg.com/features/constraints) also to avoid version differences between our dependencies in the repo.